src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int numbers;
```

Comment:
Consider renaming the variable 'numbers' to 'number' for better clarity, as it represents a single number. 🤔